Newsagents/CommandLine.java

```diff
@@ -0,0 +1,164 @@
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.util.Scanner;
+
+public class CommandLine {
+	
+	private static void listDAFuctionalityAvailable() {
+		
+		//Present Customer with Functionality Options
+		
+		System.out.println(" ");
+		System.out.println("=============================================");
+		System.out.println("Please choose ONE of the following options:");
+		System.out.println("1. Create Agent Account");
+		System.out.println("2. View ALL Agent Records");
+		System.out.println("3. Delete Agent Record by ID");
+		System.out.println("99. Close the NewsAgent Application");
+		System.out.println("=============================================");
+		System.out.println(" ");
+		
+	}
+
+	
+	private static boolean printDATable(ResultSet rs) throws Exception {
+		
+		//Print The Contents of the Full Customer Table
+		
+		System.out.println("------------------------------------------------------------------------------------------------------------------------------------");
+		System.out.println("Table: " + rs.getMetaData().getTableName(1));
+		for (int i = 1; i <= rs.getMetaData().getColumnCount(); i++) {
+			System.out.printf("%30s",rs.getMetaData().getColumnName(i));
+		}
+		System.out.println();
+		while (rs.next()) {
+			int Agent_ID = rs.getInt("Agent_ID");
+			String Agent_Name = rs.getString("Agent_Name");
+			String Agent_Area = rs.getString("Agent_Area");
+			int Agent_Pay_Rate = rs.getInt("Agent_Pay_Rate");
+			int Agent_Hours_Logged = rs.getInt("Agent_Hours_Logged");
+			System.out.printf("%30s", Agent_ID);
+			System.out.printf("%30s", Agent_Name);
+			System.out.printf("%30s", Agent_Area);
+			System.out.printf("%30s", Agent_Pay_Rate);
+			System.out.printf("%30s", Agent_Hours_Logged);
+			System.out.println();
+		}// end while
+		System.out.println("--------------------------------------------------------------------------------------------------------------------------------------");
+		
+		return true;
+		
+	}
+
+	public static void main(String[] args) {
+		
+		//Create the Database Object
+		
+		try {
+			
+			MySQLAccess dao = new MySQLAccess();
+		
+			// Configure System for Running
+			Scanner keyboard = new Scanner(System.in); 
+			String functionNumber = "-99";
+			boolean keepAppOpen = true;
+		
+			while (keepAppOpen == true) {
+			
+				//Present list of functionality and get selection
+				listDAFuctionalityAvailable();
+				functionNumber = keyboard.next();
+		
+				switch (functionNumber) {
+		
+				case "1":
+					//Get Customer Details from the User
+					System.out.printf("Enter Agents Name: \n");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.